src/plugins/dashboard/public/application/legacy_app.ts

```diff
@@ -47,6 +49,22 @@ import { DashboardListing, EMPTY_FILTER } from './listing/dashboard_listing';
 import { addHelpMenuToAppChrome } from './help_menu/help_menu_util';
 import { syncQueryStateWithUrl } from '../../../data/public';
 
+export interface DashboardListItem {
+  id: string;
+  title: string;
+  type: string;
+  description: string;
+  url: string;
+  listType: string;
+}
+
+export type DashboardListItems = DashboardListItem[];
+export type DashboardListProviderFn = () => Observable<DashboardListItems>;
+export interface DashboardDisplay {
+  hits: DashboardListItem[];
```

Comment:
nit: You already have `DashboardListItems`. Id either remove that or use that here